tests/pfcwd/test_pfcwd_all_port_storm.py

```diff
@@ -156,12 +156,23 @@ def test_all_port_storm_restore(self, duthosts, enum_rand_one_per_hwsku_frontend
         duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
         storm_hndle = storm_test_setup_restore
         logger.info("--- Testing if PFC storm is detected on all ports ---")
-        self.run_test(duthost,
-                      storm_hndle,
-                      expect_regex=[EXPECT_PFC_WD_DETECT_RE + fetch_vendor_specific_diagnosis_re(duthost)],
-                      syslog_marker="all_port_storm",
-                      action="storm")
-
+        if duthost.facts["asic_type"] in ['cisco-8000']:
+            self.run_test(
+                duthost,
+                storm_hndle,
+                expect_regex=[fetch_vendor_specific_diagnosis_re(duthost)],
+                syslog_marker="all_port_storm",
+                action="storm")
```

Comment:
Why do we need to run the storm test here? how to check the result? what's the difference between pass and failure?

Reply:
@lipxu : We makes sure that the storm is *not* detected. By removing EXPECT_PFC_WD_DETECT_RE, the loganalyzer will flag an error if the DUT pfcwd flags a storm.

Reply:
Not sure how loganalyzer flag an error if the DUT pfcwd flags a storm here?